app/src/NativeBase/Components/SkeletonLoading.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Loading screen showing animated NativeBase Skeleton components to be shown whilst a list is loading.
+ */
+
+import { Skeleton, VStack } from 'native-base'
+import React from 'react'
+
+const SkeletonLoading = () => {
+  return (
+    <>
```

Comment:
Tiny thing, but we don't need the React fragment here (`<>` and `</>`) -- [that's only needed when we don't have a single top-level component in some JSX we're rendering](https://legacy.reactjs.org/docs/fragments.html), whereas here the `VStack` does that job.

Reply:
Good catch, forgot to move during refactoring to own component file.

Reply:
Done!